fedimint-client/src/lib.rs

```diff
@@ -768,6 +768,33 @@ impl Client {
     pub async fn get_config(&self) -> &ClientConfig {
         &self.inner.config
     }
+
+    /// Balance available to the client for spending
+    pub async fn get_balance(&self) -> Amount {
```

Comment:
If this is available on the `Client` can we also remove `MintClientExt::total_amount`?

Reply:
Thx, fixed :)